translations-script/export.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+const fs = require('fs');
+const _ = require('lodash');
+const Excel = require('exceljs');
+
+function process(ws, path, enObj, gaObj) {
+  if (!_.isObject(enObj)) {
+    const enVal = enObj.replace(/\n/g, '\\n');
+    const gaVal = _.get(gaObj, path, '');
+    ws.addRow({
+      context: path,
+      english: enVal,
+      gaeilge: (gaVal && gaVal.replace(/\n/g, '\\n')) || ''
+    });
+    return;
+  }
+
+  Object.keys(enObj).forEach(key => {
+    process(ws, path === '' ? key : `${path}.${key}`, enObj[key], gaObj);
+  });
+}
+
+async function generateFile() {
```

Comment:
It would be good at some point to change this to not rely on hardcoded, individual files and instead use a directory or config file to know which languages to read and how to tag them.